node/node.go

```diff
@@ -121,6 +121,11 @@ func New(conf *Config) (*Node, error) {
 	}, nil
 }
 
+// Close releases resources acquired in Node constructor New.
+func (n *Node) Close() error {
+	return n.accman.Close()
+}
```

Comment:
I would also add a shutdown attempt to make sure everything is cleaned up if someone only calls Close, but not Stop.

```go
func (n *Node) Close() error {
	if err := n.Stop(); err != nil && err != ErrNodeStopped {
		return err
	}
	return n.accman.Close()
}
```

If you want to be truly correct, you could close both first, and only then return the errors, possible returning a combined error if both failed.

```go
func (n *Node) Close() error {
	var errs []error

	// Terminate all subsystems and collect any errors
	if err := n.Stop(); err != nil && err != ErrNodeStopped {
		errs = append(errs, err)
	}
	if err := n.accman.Close(); err != nil {
		errs = append(errs, err)
	}
	// Report any errors that might have occurred
	switch len(errs) {
		case 0: return nil
		case 1: return errs[0]
		default: return fmt.Errorf("%v", errs)
	}
}
```